parent/pom.xml

```diff
@@ -734,6 +734,11 @@
                 <artifactId>aws-java-sdk-core</artifactId>
                 <version>${aws.sdk.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Deps that are exposed from jdisc should be defined in container-dep-versions, and not be mentioned in parent. So move the version to container-dep-versions, and just remove this dep.